### DISCUSSION TITLE
Beacn Mix/Mix Create working with Pipewire on Linux

### DISCUSSION THREAD

**Discussion by CalebBrendel** [raleighlittles/beacn-linux#1] [General]
Hey, Raleigh. I don't know if this is too ambitious for you, but I would be glad in assisting you with this project. I'd love to help get a beta build going so people can test it. Idk if you are still interested, but let me know! Thanks man for making this repo!

**Comment by raleighlittles** [comment]
I'm ashamed to admit, but I never quite had the chance to get anywhere with this. I only had the unit for 30 days; things came up and I just didn't have time to get anywhere other than just do the wireshark capture on connection. I also lost interest after I found similar devices that worked with linux and fit my needs. I'm happy to transfer ownership of this repository to you if you'd like.

**Reply by TzZek** [reply]
What hardware / software did you decide to move forward with out of curiosity?

**Reply by CalebBrendel** [reply]
I agree with @TzZek. I honestly just want something similar to the hardware & software of the Mix Create. Could you enlighten us on what hardware/software you moved to? I would definitely probably take that route instead. I just have zero desire to be on Windows for my main machine, lol.
